version/distro/distro.go

```diff
@@ -70,6 +73,7 @@ func freebsdDistro() Distro {
 	case have("/usr/local/sbin/opnsense-shell"):
 		return OPNsense
 	case have("/usr/local/bin/freenas-debug"):
+		// TrueNAS Core runs on FreeBSD
```

Comment:
If this is an area you are familiar with: is there something we could use to distinguish TrueNAS from FreeNAS?

Should we even try, or are they treated interchangeably in forums / software packages / etc ?

Reply:
I'm sorry, but I'm not 100% sure, I'm new to TrueNAS.  From what I can tell though, TrueNAS Core is the new name for FreeNAS as of a few months ago (https://www.truenas.com/freenas/).